ui/src/SurveyCaseSearch.js

```diff
@@ -132,10 +135,13 @@ class SurveyCaseSearch extends Component {
       paddingBottom: "10px",
     };
 
-    return (
-      <div style={{ padding: 20 }}>
+    const searchFragment = (
+      <div>
+        <Link to={`/survey?surveyId=${this.props.surveyId}`}>
+          ← Back to survey
+        </Link>
         <Typography variant="h4" color="inherit" style={{ marginBottom: 10 }}>
-          Survey: {this.props.surveyName}
+          Search Cases
```

Comment:
Would be helpful to keep the survey name somewhere on this screen

Reply:
I know... but let's put on another ticket because it was only a 1 day spike